resources/views/projects/edit/1.blade.php

```diff
@@ -30,12 +30,12 @@
             )" :model="$project" />
 
             <fieldset class="field @error('regions') field--error @enderror" x-data="enhancedCheckboxes()">
-                <legend>{{ __('Please indicate the geographical areas this project will impact. (required)') }}</legend>
-                <x-hearth-checkboxes name="regions" :options="array_filter($regions)" :checked="old('regions_impacted', $project->regions ?? [])" required />
-                <div class="stack" x-cloak>
+                <div class="flex gap-4" x-cloak>
                     <button class="secondary" type="button" x-on:click="selectAll()">{{ __('Select all') }}</button>
                     <button class="secondary" type="button" x-on:click="selectNone()">{{ __('Select none') }}</button>
                 </div>
+                <legend>{{ __('Please indicate the geographical areas this project will impact. (required)') }}</legend>
+                <x-hearth-checkboxes name="regions" :options="array_filter($regions)" :checked="old('regions_impacted', $project->regions ?? [])" required />
```

Comment:
This doesn't seem to match the design— and also, in a previous pull request we moved the 'select all' / 'select none' buttons below the checkboxes (see #835 and #849). @cherylhjli can you clarify? The designs do not appear up to date with the changes resulting from #849.

Reply:
yes the select all/select none buttons are at the bottom now!